packages/manager/apps/public-cloud/src/components/easter-eggs/easter-eggs.constants.js

```diff
@@ -0,0 +1,13 @@
+export const EASTER_EGGS_LOCAL_STORAGE_KEY = 'PUBLIC_CLOUD_DARK_MODE';
```

Comment:
Why egg`S`? It seems like there is only one and the architecture doesn't look like it's able to handle something else than the current esater egg ?

Reply:
You're right! We can probably rename it. I add this point to the next Beta release of this component.